index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio-setup-and-mobile-version-skeleton</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <header>
+    <div class="name">Carlos Herver</div>
+    <div class="button"><img src="assets/Union.png" alt="Hamburger button"> </div>
+  </header>
+  <main>
+    <div class="intro-text">
+      <div class="primary-text">Hey There. I'm Carlos</div>
+      <div class="supporting-text">I'm a Software Developer</div>
+      <div class="Description-text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</div>
+    </div>
+    <div class="social-media">
+      <div class="icon github"><img src="assets/github.svg" alt="github.com icon"></div>
+      <div class="icon linkedin"><img src="assets/Linkedin-icon.svg" alt="linkedin.com icon"></div>
+      <div class="icon angel-list"><img src="assets/angel-list.svg" alt="angel icon"></div>
+      <div class="icon twitter"><img src="assets/twitter.svg" alt="twitter.com icon"></div>
+      <div class="icon medium"><img src="assets/medium.svg" alt="medium.com icon"></div>
+    </div>
```

Comment:
Nice work with the section 👍 , it is almost ready, but I noticed some improvements that can be made;

- [ ] It would be nice to wrap the **_icons_** and the _**intro-text**_ in a `section` tag for semantic reasons 🙏
- [ ] Kindly put this `<div class="primary-text">Hey There. I'm Carlos</div>`  in `h1` tag
- [ ] Kindly put this `<div class="supporting-text">I'm a Software Developer</div>` in `h2, h3 or h4` tag
- [ ]  Kindly put this `<div class="Description-text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
          don’t hestiate to contact me.</div>` in a `p` tag

All these changes in tags make the code semantic and easy to pick up by screen readers and you would be practicing the best HTML and CSS practices.

Reply:
Great! This makes a lot of sense. I'm going to implement the section tag to line 16 as well 👍